configure.ac

```diff
@@ -607,6 +607,16 @@ case $host_os in
     ;;
 esac
 
+have_mingw='no'
+case $host_os in
+  mingw*)
+    have_mingw='yes'
+    ;;
+esac
+
+AM_CONDITIONAL([HAVE_MINGW],
+  [test "$curl_cv_native_windows" = 'yes' -a "$have_mingw" = 'yes'])
```

Comment:
This sets the variable for newer mingw versions as well, right? I mean you want to set a compiler flag for 7.0.0 and earlier but it does set it for all versions, doesn't it?

Reply:
Yes, but, AFAICT this isn't causing an issue, because newer versions set this variable to 1 by default anyway.

Reply:
Understood. I just wanted to make sure it was done like that on purpose. Thanks!